epistemix_platform/infrastructure/tests/ecr/test_simulation_runner_repository_template.py

```
@@ -572,3 +570,135 @@ def test_ecr_scan_event_rule_not_present(self, ecr_template: dict[str, Any]):
+        "Metadata": {
+          "cfn_nag": {
+            "rules_to_suppress": [{
+              "id": "W79",
+              "reason": "Explanation of why this is acceptable"
+            }]
+          }
+        }
+        """
+        import subprocess
+
+        result = subprocess.run(
```

Ruff (0.14.2)

`subprocess` call: check for execution of untrusted input

(S603)